app/src/test/java/org/mozilla/focus/browser/integration/InputToolbarIntegrationTest.kt

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.focus.browser.integration
```

Comment:
nit: missing newline between this and the license text.

Reply:
Done